packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -1,28 +1,31 @@
 import { IWizardStyles, IWizardStyleProps } from './Wizard.types';
 
 export const subwayNavWidthPx = 303;
-
 export const subwayNavPaddingPx = 48;

```

Comment:
px is a bit misleading since that's a unit type that isn't actually built in. Unless we make this a string with the unit built in let's just remove that part so the name is more accurate.